lib/pf/services/manager/pfsetvlan.pm

```diff
@@ -14,6 +14,7 @@ pf::services::manager::pfsetvlan
 use strict;
 use warnings;
 use Moo;
+use pf::config;
```

Comment:
I thought it looked better this way.

(I forgot to remove it when I refactored this code no to need it anymore...)